docs/manual/commands/shell/qshell.rst

```diff
@@ -23,21 +23,38 @@ builtin "cd" and "ls" commands act like their familiar shell counterparts:
     > ls
     layout/  widget/  screen/  bar/     window/  group/
 
-    > cd bar
+    > cd screen
+    layout/  window/  bar/  widget/
```

Comment:
This required #2688 to be merged as that adds the ability to access `widget` from the `screen`.